thirdpartyemailpassword/docs/common-customizations/signup-form/adding-fields.md

```diff
@@ -103,17 +103,31 @@ SuperTokens.init({
                   id: "country",
                   optional: true
                 }],
-__HIGHLIGHT__                handlePostSignUp: async (user, context) => {
-                    let {id, email} = user;
-                    if (context.loginType === "emailpassword") {
-                        let formFields = context.formFields;
-                        // TODO...
-                    } else {
-                        let thirdPartyAuthCodeResponse = context.thirdPartyAuthCodeResponse;
-                        /* thirdPartyAuthCodeResponse is the response from the third party login provider that contains the access / refresh tokens sent by them..*/
+            }
+__HIGHLIGHT__            override: {
+                apis: (originalImplementation) => {
```

Comment:
- Here, I think you can link to the post sign up docs (the relevant section)?
- Also, this would need to be changed for emailpassword recipe as well?